src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const resultObject = {
+    result: 0,
+  };
+
+  resultObject.add = function (a, b) {
+    return a + b;
```

Comment:
The `add` method should use the `+=` operator to add the operand to the current result.